recipe/gen_patch_json.py

```diff
@@ -14,14 +14,14 @@
 CHANNEL_NAME = "conda-forge"
 CHANNEL_ALIAS = "https://conda-web.anaconda.org"
 SUBDIRS = (
-    #"noarch",
+    "noarch",
     "linux-64",
-    #"linux-armv7l",
-    #"linux-aarch64",
-    #"linux-ppc64le",
-    #"osx-64",
-    #"win-32",
-    #"win-64",
+    "linux-armv7l",
+    "linux-aarch64",
+    "linux-ppc64le",
+    "osx-64",
+    "win-32",
+    "win-64",
```

Comment:
uhhhh - were we not shipping repo data patches?!

Reply:
Yes, my mistake on a previous PR.